_common/qiskit/execute.py

```diff
@@ -365,77 +371,31 @@ def set_execution_target(backend_id='qasm_simulator',
                     
             backend.latest_session = session
             
-        ###############################
-        # if using IBM Quantum Platform, assume the backend_id is given only
-        elif use_ibm_quantum_platform: 
-        
-            from qiskit import IBMQ
```

Comment:
Remove the pathway that uses `qiskit.IBMQ` object since it is deprecated after qiskit 1.0, see release notes [here](https://docs.quantum.ibm.com/api/qiskit/release-notes/1.0).